app/js/lib/foxbox/network.js

```diff
@@ -56,29 +56,27 @@ export default class Network {
     return Promise.resolve();
   }
 
-  get origin() {
+  get url() {
     if (this[p.local]) {
-      return this.localOrigin;
+      return this.localUrl;
     } else if (this[p.remote]) {
-      return this.tunnelOrigin;
+      return this.tunnelUrl;
     }
 
     console.error('The box is out of reach.');
-    return this.localOrigin;
+    return this.localUrl;
   }
 
-  get localOrigin() {
+  get localUrl() {
```

Comment:
__question:__ Why do we rename ```origin``` to ```URL```? I thought ```local_url``` and ```tunnel_url``` are always just ```origins``` [1] or they can contain any ```path``` part as well? It just looks a bit confusing when we create URLs with something like ```${box.local_url}\path1\path2``` - ```box.local_url``` looks more like ```box.local_origin``` or at least ```box.local_baseURL```

[1] https://tools.ietf.org/html/rfc6454#section-3.2